### PR TITLE
Rewrite list card to not be so jittery

### PR DESCRIPTION
**Pull request by rmacqueen**
Previously we were doing all sorts of craziness
in the size allocate routine. This cleans that
up and adds a width_for_height method too.

https://phabricator.endlessm.com/T13216

### CODE REVIEW

**Comment by tchx84** [comment]
@rmacqueen Like we talked on slack, min-height: 200px, is kind of big for small screens. Just try reducing the size the window, and you will see.

File: data/css/buffet.scss [hunk 1]
```diff
@@ -119,6 +120,15 @@ $title-alt-font: 'Lato Thin' !default;
         background-color: $separator-color;
     }
 
+    .CardList {
+        &__thumbnail {
+            min-height: 200px;
+        }
+        .composite .CardList__thumbnail {
```
Comment (rmacqueen) [inline data/css/buffet.scss]: Why? Isn't this traditional CSS, of which SCSS is a subset?

Comment (tchx84) [inline data/css/buffet.scss]: Is not that. is about how scss gets translated/expanded. If you don't use the ```&``` there you will get:

```  .search-results .CardList .composite .CardList__thumbnail ```

which doesn't work (and not what we want either), vs:

```  .composite .search-results .CardList .CardList__thumbnail ```

that you would get by using ```&```.

Comment (rmacqueen) [inline data/css/buffet.scss]: fixed!

**Comment by tchx84** [comment]
**CR PASS**

**Pull request merged**